### PR TITLE
refactor(tests): refactoring of all tests

### PR DESCRIPTION
**Pull request by antoniomuso**
I am refactoring all the tests!

### CODE REVIEW

**Comment by dnlup** [comment]
Closes #77

**Comment by simonecorsi** [comment]
Much cleaner!

File: tests/configuration.test.js [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+const fastify = require('fastify');
+const { default: Dats } = require('@immobiliarelabs/dats');
+const sinon = require('sinon');
+const plugin = require('../');
+const { STATSD_METHODS } = require('../lib/util');
+
+async function setup(opts) {
+    const app = fastify();
+    app.register(plugin, opts);
+    app.get(
```
Comment (simonecorsi) [inline tests/configuration.test.js:13]: Yesterday in one of my tests I've refactored all the route using `fastify.route(opts)`, if we use that approach we can externalize the `setup` function in an utily who takes an array of route configuration objects, this way we would have only one `setup` function

**Review by simonecorsi — CHANGES_REQUESTED** [review]

File: tests/configuration.test.js (hunk 1 above)
Comment (antoniomuso) [reply tests/configuration.test.js:13]: it seems a good approach!

**Pull request merged**